lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -125,36 +149,62 @@ inject_map:
 
 maximum_paths:
   kind: int
-  config_get_token_append: '/^maximum-paths (\d+)$/'
-  config_set_append: '<state> maximum-paths <num>'
+  cli_nexus:
+    config_get_token_append: '/^maximum-paths (\d+)$/'
+    config_set_append: '<state> maximum-paths <num>'
+  cli_ios_xr:
+    # BT: Is "maximum-paths" without specialization equal to "maximum-paths ebgp"?
+    config_get_token_append: '/^maximum-paths ebgp (\d+)$/'
+    config_set_append: ' <state> maximum-paths ebgp <num>'
   default_value: 1
 
 maximum_paths_ibgp:
   kind: int
   config_get_token_append: '/^maximum-paths ibgp (\d+)$/'
-  config_set_append: '<state> maximum-paths ibgp <num>'
+  config_set_append: ' <state> maximum-paths ibgp <num>'
   default_value: 1
 
 network:
   multiple: true
-  config_get_token_append: '/^network (\S+) ?(?:route-map )?(\S+)?$/'
-  config_set_append: '<state> network <network> <route_map>'
+  cli_nexus:
+    config_get_token_append: '/^network (\S+) ?(?:route-map )?(\S+)?$/'
+    config_set_append: '<state> network <network> <route_map>'
+  cli_ios_xr:
+    config_get_token_append: '/^network (\S+) ?(?:route-policy )?(\S+)?$/'
+    config_set_append: ' <state> network <network> <route_policy>'
   default_value: []
 
 next_hop_route_map:
-  config_get_token_append: '/^nexthop route-map (.*)$/'
-  config_set_append: '<state> nexthop route-map <route_map>'
+  kind: string
+  default_value: ""
+  cli_nexus:
+    config_get_token_append: '/^nexthop route-map (.*)$/'
+    config_set_append: ' <state> nexthop route-map <route_map>'
+  cli_ios_xr:
+    config_get_token_append: '/nexthop route-policy (.*)$/'
+    config_set_append: ' <state> nexthop route-policy <route_map>'
+
+next_hop_route_policy:
+  _exclude: [cli_nexus]
+  kind: string
   default_value: ""
+  config_get_token_append: '/nexthop route-policy (.*)$/'
+  config_set_append: ' <state> nexthop route-policy <route_policy>'
```

Comment:
Shouldn't this property be removed since `next_hop_route_map` configures `route_policy` for XR?